src/AggregateFunctions/AggregateFunctionState.cpp

```diff
@@ -10,6 +10,21 @@ namespace DB
 namespace
 {
 
+namespace AgrStateDocs
```

Comment:
Had to add in cpp file, because of big system of includes. They don't have quick fix and don't let to compile project